### ISSUE TITLE
Error when moving files or directories if file or directory already exists at destination.

### ISSUE THREAD

**Issue by mohafri** [linuxmint/nemo#2871]
```
 * Nemo version 5.2.0
 * Issue with windowed nemo (file manager)
 * Distribution: up to date archlinux
 * (Graphics hardware *and* driver used)
 * 64 bit

 ```

**Issue**

1. move/copy works fine when moving/copying files or directories from one drive to another: I am asked via pop-up window if the file or directory should be replaced or duplicated.
2. copying within a drive also works perfectly.
3. moving (or drag and drop) a file or directory within a drive (or directory tree) automatically creates a copy at the destination - no pop-up window appears asking to replace or duplicate.
If there are many files at the destination, I would like to know beforehand whether the file to be moved already exists or not, in order to avoid unnecessary copies.

**Steps to reproduce**

Move (or drag and drop) a file or directory within a drive (or directory tree) if file or directory already exists at destination. the file or folder is automatically moved (a copy is generated) without being asked.

**Expected behaviour**
pop-up window with request to replace or duplicate when moving, no automatic duplication without request.

Best regards
Harald

**Comment by Jeremy7701** [comment]
This problem doesn't occur in the current `nemo` stable [5.0.5] - no duplicates are created without a dialog box prompt occurring.

**Comment by mohafri** [comment]
That's right, everything works fine with nemo 5.0.5.

However, I am using the current version nemo 5.20, and the error occurs here.

I suspect the error is caused by the change in commit 436a093b655b25b6de2643005cce9316e60dc556 in nemo-file-operations.c

With kind regards
Harald

**Comment by Haelle** [comment]
Same problem here, I don't do this often, but it's quite annoying. 

It totally forbids to merge folders with the same structure !

**Comment by pojienie** [comment]
I also have this exact same problem. My use case is that I manually manage my own Tiddly Wiki file and now I can't replace old Tiddly Wiki file by simply cut-paste.

Linux distro: Void Linux
nemo version: 5.2.0

**Comment by mtwebster** [comment]
This has been fixed since 5.2.3, you need to update.

ref:
https://github.com/linuxmint/nemo/commit/78f2e09be9ce24d52cfb0c9ba1971826e5ea85fa
